src/bundle/commitments.rs

```diff
@@ -194,4 +190,42 @@ mod tests {
             "43cfaab1ffcd8d4752e5e7479fd619c769e3ab459b6f10bbba80533608f546b0"
         );
     }
+
+    fn generate_auth_bundle<FL: OrchardFlavor>(
+        bundle_type: BundleType,
+    ) -> AuthorizedBundle<i64, FL> {
+        let mut rng = StdRng::seed_from_u64(6);
+        let pk = ProvingKey::build::<FL>();
+        let bundle = generate_bundle(bundle_type)
+            .create_proof(&pk, &mut rng)
+            .unwrap();
+        let sighash = bundle.commitment().into();
+        bundle.prepare(rng, sighash).finalize().unwrap()
+    }
+
+    /// Verify that the authorizing data commitment for an Orchard Vanilla bundle matches a fixed
+    /// reference value to ensure consistency.
+    #[test]
+    fn test_hash_bundle_auth_data_for_orchard_vanilla() {
+        let bundle = generate_auth_bundle::<OrchardVanilla>(BundleType::DEFAULT_VANILLA);
+        let orchard_auth_digest = hash_bundle_auth_data(&bundle);
+        assert_eq!(
+            orchard_auth_digest.to_hex().as_str(),
+            // Bundle hash for Orchard (vanilla) generated using
```

Comment:
should add similar explanation to `test_hash_bundle_auth_data_for_orchard_zsa`

Reply:
the explanation can be: `Generated using python test vectors, commit: 0xabcd..` or similar

Reply:
The value is not generated from python, just by running it within Orchard. The benefit is to ensure the computation doesn't get inadvertently changed in future without us realising. 

So I could put the current head commit of this branch, but that would not be a commit that is in the `zsa1` branch. It would have to be added after this PR is merged into `zsa1`, with the value of that squash merge commit.